package.json

```diff
@@ -69,6 +69,7 @@
     "husky": "^0.14.3",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^22.4.0",
+    "mini-css-extract-plugin": "^0.4.0",
```

Comment:
Can remove `extract-text-webpack-plugin` from here too (up 8 lines above this one)